examples/update-existing-zone-addresses/variables.tf

```diff
@@ -0,0 +1,37 @@
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud API Key"
+  sensitive   = true
+}
+
+variable "prefix" {
+  type        = string
+  description = "Prefix to append to all resources created by this example"
+}
+
+variable "region" {
+  description = "Name of the Region to deploy into"
+  type        = string
+}
+
+variable "resource_group" {
+  type        = string
+  description = "An existing resource group name to use for this example, if unset a new resource group will be created"
```

Comment:
`if unset a new resource group will be created` -> `a new resource group will be created if null`

Reply:
Unset means by default the assigned value is `null`. I do not think anything is wrong with this line.